patches/chrome-installer-mac-signing-pipeline.py.patch

```diff
@@ -1,12 +1,34 @@
 diff --git a/chrome/installer/mac/signing/pipeline.py b/chrome/installer/mac/signing/pipeline.py
-index 898c0d4542bb3474d654e71a1e4fd5a1355e3897..dad0718097522f2346f690ae7435dbe162cbb964 100644
+index 898c0d4542bb3474d654e71a1e4fd5a1355e3897..a1a32cac6e7e23bda56411b165ff0fe84a620e91 100644
 --- a/chrome/installer/mac/signing/pipeline.py
 +++ b/chrome/installer/mac/signing/pipeline.py
-@@ -221,7 +221,6 @@ def _package_installer_tools(paths, config):
+@@ -192,12 +192,6 @@ def _package_dmg(paths, dist, config):
+         '--volname', config.app_product,
+         '--icon', os.path.join(packaging_dir, icon_file),
+         '--copy', '{}:/'.format(app_path),
+-        '--copy',
```

Comment:
why do we need to remove this?

Reply:
The `keystone_install.sh` part of the command isn't copied or called, leaving it in caused a build break during my testing.

Reply:
where did it break? Can we remote it later or use a dummy script just to satisfy the copy? We need to find other ways to avoid patches like this

Reply:
OK so we can do a touch (to create blank files for `keystone_install` and `chrome_dmg_background.png`) and then remove it before packaging